access/types/stdasync.py

```diff
@@ -362,7 +361,7 @@ def _acceptSubmission(request, course, exercise, post_url, sdir):
     '''
     uids = get_uid(request)
     attempt = int(request.GET.get("ordinal_number", 1))
-    container_flag = settings.CONTAINER_MODE and "container" in exercise
+    container_flag = "container" in exercise
```

Comment:
The `container_flag` variable looks unnecessary now that only container mode is supported and there is no alternative. The code already calls the `_requireContainer` function before using the `container` field from the exercise config.yaml, which is good. There is no need to make useless if checks (`if container_flag:`) that are always supposed to be True.

If the `container` field is missing from the exercise config.yaml and the `actions` field exists, there could be a clear error to the user (something like "the actions field must be replaced with the container field because the chroot sandbox is no longer supported"). This could be done by raising a `ConfigError` in the `_requireContainer` function.

Reply:
Done!